### DISCUSSION TITLE
Looking for font u8g2_font_04b_03b_tr

### DISCUSSION THREAD

**Discussion by Fishbone69** [olikraus/u8g2#2211] [General]
This font is in the reference material and I would like to use it, but for some reason u8g2.setfont doesn't recognize it.  Was this font removed from the library?  Is it something I have to load separately?

Thanks for any help.

**Comment by olikraus** [comment]
It should be included in the latest u8g2 release

**Reply by Fishbone69** [reply]
Thanks, man...ur the best.  My digital watch software is almost finished - u8g2 has really helped.  I'll post some pics a bit later...should I do it here in discussions?

**Comment by olikraus** [comment]
Pictures are always welcome :-)
We can change category to "show & tell" :-D
